src/components/visualization-list-menu-item/visualization-list-menu-item.scss

```diff
@@ -0,0 +1,22 @@
+@import "../../styles/variables.scss";
+@import "../../styles/mixins.scss";
+
+visualization-list-menu-item {
+
+  .folder-icon {
+    fill: $medium-grey;
```

Comment:
could you sassify this to be
```
  .folder-icon {
    fill: $medium-grey;
    &:hover {
        fill: $almost-black;
    }
}

Reply:
Sure. For the record, this was not really code that we touched - we just moved it.  😁